android/build.gradle

```diff
@@ -27,6 +27,10 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
+  if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
```

Comment:
I think this could be an issue on < AGP 7.3, since that's when Google stated you can start using the namespace DSL property (correct me if im wrong though). Maybe it's better to just change 7 -> 8?

Reply:
could also check for major version >= 7 and minor >= 3? would be more exact that way